### PR TITLE
Added some Query for UPDATE DELETE and make use of transaction

### PR DESCRIPTION
**Pull request by jubaer919**
# Highlight 🔍 

- Make sure the project uses the correct Gitflow.
- Add a column `species` of type `string` to my animals table.
- INSERT some given data INTO my animals table.
- Check every single query to pass the project. Here is the Screen shot:

et-clinic-database/assets/60

255197/f1677522-af67-44fa-8a

93-a48f8cec0782)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,6 +12,13 @@ VALUES
...
 (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
-(4, 'Devimon', '2017-05-12', 5, true, 11);
+(4, 'Devimon', '2017-05-12', 5, true, 11),
+(5, 'Charmander', '2020-02-08', 0, false, 11),
+(6, 'Plantmon', '2021-11-15', 2, true, 5.7),
+(7, 'Squirtle', '1993-04-02', 3, false, 12.13),
+(8, 'Angemon', '2005-06-12', 1, true, 45),
+(9, 'Boarmon', '2005-06-07', 7, true, 20.4),
+(10, 'Blossom', '1998-10-13', 3, true, 17),
+(11, 'Ditto', '2022-05-14', 4, true, 24);
```
Comment (usorfaitheloho) [inline data.sql]: You did well with the implementation of this project, kindly note that the weight values for `Chamander, Plantmon,Squirtle and Angemon` should have negative values as it is stated in the requirement 😄

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,51 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+
+SELECT neutered, COUNT(escape_attempts) as escape_attempts_count
+FROM animals
+GROUP BY neutered
+ORDER BY escape_attempts_count DESC;
+
+SELECT name, escape_attempts
+FROM animals
+WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
+
```
Comment (usorfaitheloho) [inline queries.sql]: Kindly note that the query for the maximum and minimum weight of each animal should be similar to this :

`   SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
   GROUP BY species;
`

Comment (usorfaitheloho) [reply queries.sql]: Also, the query for  "What is the average number of escape attempts per animal type of those born between 1990 and 2000?" should be similar to this :
`   SELECT species, AVG(escape_attempts) FROM animals
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/12/31'
   GROUP BY species;

`

Kindly write the query to produce accurate results 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @jubaer919  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Used database transactions. ✔️ 
> * Modify and delete data in SQL. ✔️ 

> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
# Hi @jubaer919 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct queries for each question✔
- Correct use of Git Flow ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**